README.md

```diff
@@ -7,23 +7,27 @@
   - https://github.com/HiveFunder/support-module-svc
   - https://github.com/HiveFunder/updates-service
 
+## Setup Instructions
+To seed data for this project, first run `node db/largeDataGenAndSeed.js`, then run the bash shell script `./db/mongoSeed.sh`, both of which from the project root.
+If your laptop has an SSD, it should take no more than 30 minutes to generate and seed everything!
+
 ## CRUD Documentation
 
 ### Data Shape
-  - Campaign: Campaign name, description, author, currency, pledge amount, pledge goal, backers, end date, type, location
+  - Campaign: Campaign name, description, author, username, country, pledge amount, pledge goal, backers, end date, type
   - Pledge: username, pledge amount, date
 
 ### Page Endpoint
-/:author/:campaignId/
+/:user/:campaignId/
```

Comment:
Is this a functional change, or a conventional change? There's no problem either way, I just want to make sure I'm on board with my conventions.

Reply:
Conventional change - I realized Author refers to a person's name which would include spaces, whereas User refers to a username and would make more sense. This is how kickstarter does it, but my endpoints don't conform to this yet